### DISCUSSION TITLE
Not working EvaluateExpressionAsync

### DISCUSSION THREAD

**Discussion by seherkaya** [cefsharp/CefSharp.OutOfProcess#12] [General]
When I want to run EvaluateExpressionAsync in Out Of process project, I cannot run it because MainFrame is not created when ChromiumWebBrowser first occurs.

**Comment by amaitland** [comment]
Please read https://idownvotedbecau.se/itsnotworking/

Insufficient information to provide any guidance.

**Comment by seherkaya** [comment]
I want to run different versions of cefsharp in two different processes.
In the Cefsharp version I use in the out of process, I run the EvaluateExpressionAsync method to run some javascript commands in the background. But ChromiumWebBrowser's MainFrame sometimes occurs and sometimes it doesn't. What should I do ?
